text/0122-sdk-hub-scope-merge.md

```diff
@@ -0,0 +1,475 @@
+- Start Date: 2023-11-07
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/122
+- RFC Status: draft
+
+# Summary
```

Comment:
Since this RFC is no longer being updated, we're writing down how this works and collecting learnings etc. here: https://develop.sentry.dev/sdk/hub_and_scope_refactoring/